.github/workflows/tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
           ini-values: display_errors=off, log_errors=on
-          extensions: :xdebug
+          extensions: :xdebug, ctype, dom, gd, iconv, ldap, mbstring, reflection, session, simplexml, spl, xml, zlib, memcache, pdo_sqlite, bcmath
```

Comment:
somewhy ext-ldap is missing for 5.3 (you need to click on "Setup PHP 5.3" to see output):
- https://github.com/glensc/php-zf1s/runs/1597658212?check_suite_focus=true

```
==> Setup Extensions
✓ ctype Enabled
✓ dom Enabled
✓ gd Enabled
✓ iconv Enabled
✗ ldap Could not install ldap on PHP 5.3.29
✓ mbstring Enabled
...
```

Reply:
btw, some earlier builds have succeeding as the cache is used and composer install step skipped:
- https://github.com/glensc/php-zf1s/runs/1597629716?check_suite_focus=true

Reply:
reported: https://github.com/shivammathur/setup-php/issues/386

Reply:
https://github.com/shivammathur/setup-php/issues/386 is fixed